apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx

```diff
@@ -1,32 +1,71 @@
 import React from 'react';
 import Spinner from '../pd/workshop_dashboard/components/spinner';
-import EscalatedSubmissions from './EscalatedSubmissions';
+import PeerReviewSubmissions from './PeerReviewSubmissions';
 
 const PeerReviewDashboard = React.createClass({
```

Comment:
given that createClass is deprecated pending complete removal, now might be a good time to update this to an ES6 class